ERCS/erc-7515.md

```diff
@@ -0,0 +1,124 @@
+---
+eip: 7515
+title: NFT Alternative Text in Metadata
+description: Populate alternative text field in NFT Metadata with image description for screen reader.
+author: Mona Rassouli (@DecoratedWings)
+discussions-to: https://ethereum-magicians.org/t/proposal-for-informational-eip-simple-accessibility-recommendation/14639
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-08-15
+---
+
+## Abstract
+
+This ERC advocates for the consistent inclusion of alternative text in NFT metadata. Marketplaces, creators, and decentralized applications should adopt this best practice to improve accessibility. When rendering NFT images on frontend interfaces, these platforms can leverage the `alt` attribute effectively.
```

Comment:
Hi @DecoratedWings thanks for the draft. can you help me see the necessity of not using `description` but having another `alt`?

Reply:
Hi @xinbenlv, 
Absolutely, thank you for the question. Typically I've noticed that the description field in the metadata is used more to describe aspects of the collection or longer details even for that particular token. Alternative Text should be limited to a specific description of what is actually in the image. 

I grabbed an example at random from opensea to help illustrate: https://opensea.io/assets/ethereum/0x5463612c30cd4deac35af4a930c6f997dba24ad1/287

In this case, the description field was used to explain the collection. Token 287 has the following metadata: https://app.bueno.art/api/contract/24vXZhPEzz593uVhdyoCE/chain/1/metadata/287

If it were to have the `alt` field it would actually described the image of token 287, stating something like: "An abstract portayal of a woman with brown hair and white shirt playing a game of chess. There is a black and white patterned background with pink flowers along the left frame."